pynest/nest/lib/hl_api_connections.py

```diff
@@ -27,44 +27,84 @@
 from .hl_api_nodes import Create
 from .hl_api_info import GetStatus
 from .hl_api_simulation import GetKernelStatus, SetKernelStatus
-
 import numpy
 
+
 @check_stack
 @deprecated(alt_func_name='GetConnections')
-def FindConnections(source, target=None, synapse_model=None, synapse_type=None):
-    """
-    Return an array of identifiers for connections that match the
-    given parameters. Only source is mandatory and must be a list of
-    one or more nodes. If target and/or synapse_model is/are given,
-    they must be single values, lists of length one or the same length
-    as source. Use GetStatus()/SetStatus() to inspect/modify the found
+def FindConnections(source, target=None, synapse_model=None,
+                    synapse_type=None):
+    """Return an array of identifiers for connections that match the
+    given parameters.
+
+    .. note:: Deprecated
+        FindConnections() is deprecated and will be removed in the future.
+        Use GetConnections() instead.
+
+    If target and/or synapse_model is/are given, they must be single
+    values, lists of length one or the same length as source.
+    Use GetStatus()/SetStatus() to inspect/modify the found
     connections.
 
-    Note: FindConnections() is deprecated and will be removed in the future.
-          Use GetConnections() instead.
-
-    Note: synapse_type is alias for synapse_model for backward compatibility
+    Parameters
+    ----------
+    source : list
+        Source GIDs, only connections from these
+        pre-synaptic neurons are returned
+    target : list, optional
+        Target GIDs, only connections to these
+        post-synaptic neurons are returned
+    synapse_model : str, optional
+        Only connections with this synapse model are returned
+    synapse_type : str, optional
+        Only connections with this synapse type are returned
```

Comment:
I'd add the fact that this is the same as `synapse_model`. On the other side, the function is deprecated since long and can as well be deleted right after this PR is merged.